src/main/java/core/basesyntax/Car.java

```diff
@@ -1,5 +1,70 @@
 package core.basesyntax;
 
-public class Car {
+public class Car implements Cloneable {
 
+    private Engine engine;
+    private final String model;
+    private String color;
+    private String owner;
+    private int yearOfProduction;
+
+    public Car(Engine engine, String model, String color, String owner, int yearOfProduction) {
+        this.engine = engine;
+        this.model = model;
+        this.color = color;
+        this.owner = owner;
+        this.yearOfProduction = yearOfProduction;
+    }
+
+    @Override
+    public Car clone() {
+        try {
+            Car clonedCar = (Car) super.clone();
+            clonedCar.setEngine(this.engine.clone());
+            return clonedCar;
+        } catch (CloneNotSupportedException e) {
+            throw new RuntimeException("Could not clone car", e);
+        }
+    }
+
+    public String toString() {
+        return new StringBuilder("Car: model = ").append(this.model)
```

Comment:
let's not use StringBuilder in toString(). Default overrided implementation is good enough